roles/openshift_logging/library/openshift_logging_facts.py

```diff
@@ -249,7 +249,7 @@ def facts_for_secrets(self, namespace):
     def facts_for_sccs(self):
         ''' Gathers facts for SCCs used with logging '''
         self.default_keys_for("sccs")
-        scc = self.oc_command("get", "scc", name="privileged")
```

Comment:
You sure? Just installed v3.7.0 and it's there:
```
oc ve[centos@ip-10-0-0-166 ~]$ oc version
oc v3.7.0+7ed6862
kubernetes v1.7.6+a08f5eeb62
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://api-internal.engine.getupcloud.com:443
openshift v3.7.0+7ed6862
kubernetes v1.7.6+a08f5eeb62
[centos@ip-10-0-0-166 ~]$ oc get scc
NAME               PRIV      CAPS      SELINUX     RUNASUSER          FSGROUP     SUPGROUP    PRIORITY   READONLYROOTFS   VOLUMES
anyuid             false     []        MustRunAs   RunAsAny           RunAsAny    RunAsAny    10         false            [configMap downwardAPI emptyDir persistentVolumeClaim projected secret]
hostaccess         false     []        MustRunAs   MustRunAsRange     MustRunAs   RunAsAny    <none>     false            [configMap downwardAPI emptyDir hostPath persistentVolumeClaim projected secret]
hostmount-anyuid   false     []        MustRunAs   RunAsAny           RunAsAny    RunAsAny    <none>     false            [configMap downwardAPI emptyDir hostPath nfs persistentVolumeClaim projected secret]
hostnetwork        false     []        MustRunAs   MustRunAsRange     MustRunAs   MustRunAs   <none>     false            [configMap downwardAPI emptyDir persistentVolumeClaim projected secret]
nonroot            false     []        MustRunAs   MustRunAsNonRoot   RunAsAny    RunAsAny    <none>     false            [configMap downwardAPI emptyDir persistentVolumeClaim projected secret]
privileged         true      [*]       RunAsAny    RunAsAny           RunAsAny    RunAsAny    <none>     false            [*]
restricted         false     []        MustRunAs   MustRunAsRange     MustRunAs   RunAsAny    <none>     false            [configMap downwardAPI emptyDir persistentVolumeClaim projected secret]
[centos@ip-10-0-0-166 ~]$
```

Also it appears on API as a valid shortname:

```
[centos@ip-10-0-0-166 ~]$ curl -sk https://api.engine.getupcloud.com/apis/security.openshift.io/v1/ | grep scc -C2
      ],
      "shortNames": [
        "scc"
      ]
    }
[centos@ip-10-0-0-166 ~]$
```

Reply:
you are right，